src/Raven.Studio/typescript/viewmodels/manage/trafficWatch.ts

```diff
@@ -13,33 +13,73 @@ import virtualColumn = require("widgets/virtualGrid/columns/virtualColumn");
 import recentQueriesStorage = require("common/storage/savedQueriesStorage");
 import queryCriteria = require("models/database/query/queryCriteria");
 import databasesManager = require("common/shell/databasesManager");
+import TrafficWatchHttpChange = Raven.Client.Documents.Changes.TrafficWatchHttpChange;
+import trafficWatchQueriesDialog from "viewmodels/manage/trafficWatchQueriesDialog";
+import app = require("durandal/app");
 
 type trafficChangeType = Raven.Client.Documents.Changes.TrafficWatchChangeType | Raven.Client.ServerWide.Tcp.TcpConnectionHeaderMessage.OperationTypes; 
 
 class runQueryFeature implements columnPreviewFeature {
-    install($tooltip: JQuery, valueProvider: () => any, elementProvider: () => any, containerSelector: string) {
+    install($tooltip: JQuery, valueProvider: () => any, elementProvider: () => any, containerSelector: string): void {
         $tooltip.on("click", ".run-query", () => {
-            const value = valueProvider();
-
+            let value = valueProvider();
```

Comment:
why value is let right, now?

Reply:
set back to const (forgot to set it back when done playing w/ the code)